pkg/actors/internal/placement/client/client.go

```diff
@@ -235,6 +235,11 @@ func (c *Client) Recv(ctx context.Context) (*v1pb.PlacementOrder, error) {
 			errors.Is(err, io.EOF) {
 			log.Infof("Placement service is closed, reconnecting...")
 			errCh := make(chan error, 1)
+			select {
+			case <-ctx.Done():
+			case <-time.After(time.Second):
```

Comment:
Why 1s? Is it enough?

Time sleep may hide the root cause, is it only caused by network issue?

Reply:
It's an arbitrary number which is "long enough" to not spam, and "short enough" to log that daprd is doing something. All this number is doing is setting the interval for when we try to connect to the placement leader again. Without is, we will have a reconnect error spam every few ms. This produces 1000s of log lines needlessly.

This is typically caused when the placement pod is restarted for whatever reason, e.g. being scheduled to a new node. It should be considered normal operation stuff.

Reply:
Indeed, there may be other issues going on though!

Reply:
> It's an arbitrary number which is "long enough" to not spam, and "short enough" to log that daprd is doing something. All this number is doing is setting the interval for when we try to connect to the placement leader again. Without it, we will have a reconnect error spam every few ms. This produces 1000s of log lines needlessly.
> 
> This is typically caused when the placement pod is restarted for whatever reason, e.g. being scheduled to a new node. It should be considered normal operation stuff.

Please add those into PR description.